backend/index.js

```diff
@@ -0,0 +1,13 @@
+import app from "./app.server";
+import officeFactory from "./office.factory";
+
+const PORT = process.env.PORT || 8080;
+const HOST = "0.0.0.0";
```

Comment:
Move it to `app.config.js`, makes sense?

Reply:
done!